src/function.rs

```diff
@@ -92,6 +94,14 @@ pub fn os_family(_context: &FunctionContext) -> Result<String, String> {
   Ok(target::os_family().to_string())
 }
 
+pub fn invocation_directory(context: &FunctionContext) -> Result<String, String> {
+  if let Some(dir) = context.invocation_directory {
+    Ok(dir.to_string())
+  } else {
+    Err(String::from("error getting invocation directory"))
```

Comment:
I'm not super happy with this error message right now.  The primary reason for this error would be an inability to decode the utf-8 of the original working directory, but it's not clear.  Feedback/thoughts welcome.

Reply:
I think that a really explicit error message is good, perhaps "invocation directory path not UTF-8, `invocation-directory` cannot be used"

Reply:
Also, you can use [Option::ok_or_else](https://doc.rust-lang.org/std/option/enum.Option.html#method.ok_or_else) to turn `None` into an error.

Reply:
Yeah - I don't disagree.  The only thing that gives me pause is that _where the error occurred_ is pretty far (in code) from where this string is - which makes me think that they could fall out of sync.  For example, one day in the future there could be a different reason that `invocation_directory` can't be used, and it would be easy to forget to update this message.

What do you think about just having `invocation_directory` be passed along as a `Result<String, String>`?  That way, the error string is constructed close to the error itself.

Alternatively, `just` could refuse to run when invoked from a non-utf8 path, but that seems rather draconian.